### DISCUSSION TITLE
veeValidate Form breaks when using <Form> and not native html. all examples have <form>

### DISCUSSION THREAD

**Discussion by orenmizr** [unovue/shadcn-vue#477] [Q&A]
it simply does not call onSubmit handler but it does validation. 
vee validates's composable works for but i was trying to test vee validates's Wrapper and @zernonia form elements.

**Comment by Saeid-Za** [comment]
If this is an issue, could you please open one and provide a reproduction repo?
Thanks!

**Comment by zernonia** [comment]
The `onSubmit` will only be trigger if there's no error in validation. Do check that condition @orenmizr  😁
Checkout `submitForm` on [VeeValidate docs](https://vee-validate.logaretm.com/v4/api/form#form-component)

**Comment by orenmizr** [comment]
it was tested with valid conditions. : ) 
shadcn-vue has abstractions for useForm  and other veeValidate components. i didn;t understand why in the examples we use form components with native  form tag. once i used either shadcn-vue's form components or native with hook - it worked. when i copied the example it did not work for me. i figured because it didn;'t share the formId and context correctly when they are mixed.
